workflow/scripts/predict.py

```diff
@@ -265,7 +273,8 @@ def main():
     print("Writing output files...")
     all_putative = pd.concat(all_putative_list)
     all_putative["CellType"] = args.cellType
-    all_putative["hic_contact_squared"] = all_putative["hic_contact"] ** 2
```

Comment:
this would throw an exception if hic_dir is not passed in. Fix by ensuring we only calculate this if hi-c provided